testdata/project-v2-addon/go.mod

```diff
@@ -6,8 +6,10 @@ require (
 	github.com/go-logr/logr v0.1.0
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.8.1
-	k8s.io/apimachinery v0.17.2
-	k8s.io/client-go v0.17.2
-	sigs.k8s.io/controller-runtime v0.5.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200415210853-85eb326a6add
+	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
+	k8s.io/apimachinery v0.18.2
+	k8s.io/client-go v0.18.2
+	sigs.k8s.io/controller-runtime v0.6.0
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200507203943-d901af431e3a
+	sigs.k8s.io/structured-merge-diff v1.0.1-0.20191108220359-b1b620dd3f06 // indirect
```

Comment:
Indirect deps which were updated with sigs.k8s.io/kubebuilder-declarative-pattern

Reply:
Does this change belong to this PR? I made the same question in other PR so we can discuss it in only one place and leave a link to it in the rest of the places.

Reply:
Yes. I am waiting for an LGTM in https://github.com/kubernetes-sigs/kubebuilder/pull/1504 to be able to rebase all open pre.

Reply:
solved